# A single point in a desired trajectory

float64[6] eta_desired    # Desired position and orientation
float64[6] nu_desired     # Desired linear and angular velocities
